src/__tests__/touchable-test.tsx

```diff
@@ -38,6 +38,9 @@ test('<a> element is rendered when "to" prop is used and no Link component injec
+          </style>
```

Comment:
🤔 this could potentially be duplicated more and more in the future. I'd extract it to a constant or something more maintainable

Reply:
what do you mean? this is an inline snapshot, it's autogenerated by jest.
Anyway, I'm going to remove this from tests

Reply:
done